source/_page_templates/ability_template.rst

```diff
@@ -0,0 +1,58 @@
+.. [insert ability name here, lowercase, one word]:
+############
+ABILITY NAME
+############
+
+Description
+###########
+
+Paste description here. Insert gif here |ABILNAMEHERE1|
+
+
+Usage
+#####
+Below are the steps to use this ability
+
+Insert in-game gif here (add another substition)
+
+Function 1: [] then [] then []
```

Comment:
This can just be the copy-pasted usage.

Reply:
Maybe add in some tip sections here?

Reply:
That is what the note was for

and the copy pasted Usage is often insufficient to explain certain functions like Shifting Between Ice and Water on SurgeWall, or using Combos